src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBlockControl/TextBlock_IsTextTrimmedChanged.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<UserControl x:Class="Uno.UI.Samples.Content.UITests.TextBlockControl.TextBlock_IsTextTrimmedChanged"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 mc:Ignorable="d"
+			 d:DesignHeight="800"
+			 d:DesignWidth="800">
+
+	<StackPanel Width="80">
+		<TextBlock IsTextTrimmedChanged="TextBlock_IsTextTrimmedChanged_Event"
+				   TextWrapping="Wrap"
+				   MaxLines="3"
+				   TextTrimming="CharacterEllipsis"
+				   x:Name="MyTextBlock"
+				   Text="Lorem ipsum dolor sit amet consectetur. Porta habitasse iaculis non egestas posuere in tincidunt. Risus nunc a  urna est nunc ac mattis eget. Faucibus ullamcorper ipsum faucibus in tristique lacus. Justo cursus arcu ut tristique." />
+	</StackPanel>
+</UserControl>
```

Comment:
(there is no visual indication here... update the result to a textblock?)

but this could just a runtime test to avoid wasting qa time

Reply:
that said, you should keep an ui test version
where the stack-panel width is bound to a slider so we can validate edge cases (the ones near the tipping point)
with a textbbock bound to IsTextTrimmed